src/java/org/apache/cassandra/db/partitions/BTreePartitionUpdate.java

```diff
@@ -261,6 +291,19 @@ public int dataSize()
                 + holder.staticRow.dataSize());
     }
 
+    // FIXME review
+    /**
+     * The size of the data contained in this update.
+     *
+     * @return the size of the data contained in this update.
+     */
+    @Override
+    public long unsharedHeapSize()
+    {
+        return BTree.<Row>accumulate(holder.tree, (row, value) -> row.unsharedHeapSize() + value, 0L)
+                + holder.staticRow.unsharedHeapSize();
+    }
+
```

Comment:
For Review:
`accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. The implementation has moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.